woodwork/tests/accessor/test_statistics.py

```diff
@@ -1254,24 +1253,21 @@ def test_percentile_func_usage(mock_percentile, describe_df):
         numeric_data.ww.describe()
         assert not mock_percentile.called
 
-    # Test numeric with non-nullable ltypes
-    numeric_data = describe_df[["numeric_col"]].dropna()
-    for ltype in non_nullable_numeric_ltypes:
-        numeric_data.ww.init(
-            logical_types={"numeric_col": ltype},
-            semantic_tags={"numeric_col": "custom_tag"},
-        )
-        numeric_data.ww.describe()
-        assert mock_percentile.called
 
+@patch.object(sys.modules["woodwork.statistics_utils._get_describe_dict"], "percentile")
+@pytest.mark.parametrize("non_nullable_numeric_type", [Integer, Age])
```

Comment:
one more thing! 

Can you add [Double, IntegerNullable, AgeNullable, AgeFractional] to the parametrization? It's a small thing, but we'll want to know that it's not just the type that is defining whether or not percentile gets used; it's whether nans are present. I would also update the test names to `test_percentile_func_not_called_with_nans` and `test_percentile_func_called_without_nans`